ALoop/donations.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- ticker: ALoop
+  announcement_url: https://ablegamers.org/
+  recipient:
+    name: The AbleGamers Charity
+    address: 179 E. Burr Blvd, Suite Q Kearneysville, WV 25430
+    tel: +1 703 891 9017
+    contact: inquiries@ablegamers.com
+  ada: 20
+  usd: 25.00
+  timestamp: 1626313680
+  files:
+     - Aloop/2021-07-18-ABG.png
```

Comment:
The file name seems to be case-sensitive. Please just do the Ticker full upper case.
`ALoop/2021-07-18-ABG.png
vs
Aloop/2021-07-18-ABG.png
`